src/kirby/components/card/card-header/card-header.component.tns.html

```diff
@@ -1 +1,5 @@
-<Button text="card-header works!" class="btn btn-primary"></Button>
\ No newline at end of file
+<StackLayout class="header">
+  <Label class="title" *ngIf="title" text="{{title}}"></Label>
+  <Label class="subtitle" *ngIf="subtitle" text="{{subtitle}}"></Label>
```

Comment:
```suggestion
  <Label class="subtitle" *ngIf="subtitle" [text]="subtitle"></Label>
```

Reply:
Ups ja, for meget copy and paste med hovedet under armen, fikset nu